src/flag.h

```diff
@@ -0,0 +1,49 @@
+#ifndef FLAG_H
+#define FLAG_H
+
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#define DEFAULT_PORT_NUMBER 8081
+#define DEFAULT_WEBROOT "./www"
+
+static int get_port_number(char *arg_port)
+{
+    int port_number = strtol(arg_port, &arg_port, 10);
+    if (port_number <= 0 || port_number > 65535) {
+        port_number = DEFAULT_PORT_NUMBER;
+    }
+    return port_number;
+}
+
+struct cmd_config {
+    unsigned short int port;
+};
+
+static struct cmd_config *get_config_by_cmd(int argc, char **argv)
+{
+    int cmdopt = 0;
+    struct cmd_config *cfg =
```

Comment:
It is not necessary to cast `(struct cmd_config *)`. That is, you can safely remove `(struct cmd_config *)`.

Reply:
I have removed it.  Please check. Thank you!